Automattic/wp-calypso
# Fix new onboarding flow tests

#### Changes proposed in this Pull Request

* This PR re-enables and fixes two onboarding flow tests

#### Testing instructions

* Ensure `Sign up for a free WordPress.com site via the new onboarding flow` and `Sign up for an account only (no site) then add a site via new onboarding flow` tests pass
